core/src/main/resources/jenkins/management/AdministrativeMonitorsConfiguration/config.groovy

```diff
@@ -34,7 +34,8 @@ f.section(title: _("Administrative monitors configuration")) {
         f.entry(title: _("Enabled administrative monitors")) {
             p(_("blurb"))
             table(width: "100%") {
-                for (AdministrativeMonitor am : AdministrativeMonitor.all()) {
+                for (AdministrativeMonitor am : new ArrayList<>(AdministrativeMonitor.all())
+                        .sort({ o1, o2 -> o1.getDisplayName() <=> o2.getDisplayName() })) {
```

Comment:
@Dohbedoh Could you give me some reference to this operator `<=>`? Didn't use it before. :eyes: